addons/hr_recruitment_survey/data/survey_demo.xml

```diff
@@ -2,7 +2,7 @@
 <odoo><data noupdate="1">
     <record id="survey_recruitment_form" model="survey.survey">
         <field name="title">Recruitment Form</field>
-        <field name="stage_id" ref="survey.stage_in_progress"/>
+        <field name="state" eval="'open'" />
```

Comment:
You can simply use <field name="state">open</field> . No need to explicitly call en eval.